src/gallery/gallery.macros.html

```diff
@@ -1,12 +1,25 @@
 {#
-  * @overview Macros for Gallery carousel that use Flickity
+  * @overview Macros for Gallery carousels that use Flickity
 #}
 
 {#
   * Renders a Gallery carousel with custom Flickity options
+  * @param {boolean} Setting to display a gallery of images (optional, default: false)
 #}
-{% macro gallery(autoplay=false, arrows=true, dots=true, draggable=false, wraparound=true, class=null, id=null) %}
-<div class="gallery js-gallery{% if class %} {{ class }}{% endif %}"{% if id %} id="{{ id }}"{% endif %} data-options='{ "autoPlay": {{ autoplay }}, "prevNextButtons": {{ arrows }}, "pageDots": {{ dots }}, "draggable": {{ draggable }}, "wrapAround": {{ wraparound }} }'>
+{% macro gallery(image_gallery=false, autoplay=false, arrows=true, dots=true, draggable=false, wraparound=true, class=null, id=null) %}
+<div class="gallery{% if image_gallery %} gallery--images js-gallery--images{% else %} js-gallery{% endif %}{% if class %} {{ class }}{% endif %}"{% if id %} id="{{ id }}"{% endif %} data-options='{ "autoPlay": {{ autoplay }}, "prevNextButtons": {{ arrows }}, "pageDots": {{ dots }}, "draggable": {{ draggable }}, "wrapAround": {{ wraparound }} }'>
   {{ caller() }}
 </div>
 {% endmacro %}
+
+{#
+  * Renders a Gallery image for use in a carousel
+  * @param {string} Gallery image imgix path
+  * @param {string} Image description alt text (optional)
+  * @param {string} Class name for imgix fluid targeting (optional, default: 'imgix-fluid')
+  * @param {string} Class name for style or JS targeting (optional)
+  * @param {string} ID for JS targeting (optional)
+#}
+{% macro gallery_image(image_path, image_description=null, imgix_class='imgix-fluid--gallery', class=null, id=null) %}
```

Comment:
I'd like to get input from the team on what to call these macros. My brain instinctively wants to call this a gallery slide. Maybe it is clear enough to others.